src/Number.jl

```diff
@@ -65,9 +65,9 @@ function parse_number(s, i)
     end
 
     # int
-    while c in UInt8('0'):UInt8('9')
+    while (decimal = c - UInt8('0')) < 0x10
```

Comment:
Thanks @KristofferC !
(I changed it to `< 10` 🙃)

Reply:
https://github.com/samoconnor/LazyJSON.jl/blob/e8d2494c4df9c3c344c081cf9e706584ad246e4c/test/runtests.jl#L103-L106

Reply:
Oops, thanks.

Perhaps better with `Int8(10)`?